draft-brunstrom-taps-impl.md

```diff
@@ -427,7 +427,7 @@ Protocols that provide the framing (such as length-value protocols, or protocols
 
 ### Sending Messages
 
-The effect of the application sending a Message is determined by the top-level protocol in the established Protocol Stack. That is, if the top-level protocol provides an abstraction of framed messages over a connection, the application will be able to send multiple Messages on that connection, even if the framing protocol is built on a byte-stream protocol like TCP.
+The effect of the application sending a Message is determined by the top-level protocol in the established Protocol Stack. That is, if the top-level protocol provides an abstraction of framed messages over a connection, the receiving application will be able to obtain multiple Messages on that connection without supplying a deframer, even if the framing protocol is built on a byte-stream protocol like TCP.
```

Comment:
The point of this paragraph was to describe how you can send multiple message using a framer. I don't think we should add "without a deframer", since the point was to describe the framer.

Reply:
I get it now, agreed